Sources/SpelledPitch/PitchSpeller/Wetherfield/FlowNetwork.swift

```diff
@@ -60,11 +60,25 @@ extension FlowNetwork {
         }
     }
     
-    mutating func mask (_ adjacencyScheme: GraphScheme<Node>) {
-        for edge in edges where !adjacencyScheme.contains(GraphScheme<Node>.Edge(edge.a, edge.b)) {
+    mutating func mask <Scheme: UnweightedGraphSchemeProtocol> (_ adjacencyScheme: Scheme) where
+        Scheme.Node == Node
+    {
+        for edge in edges where !adjacencyScheme.contains(from: edge.a, to: edge.b) {
             remove(edge)
         }
     }
+    
+    mutating func mask <Scheme: WeightedGraphSchemeProtocol> (_ weightScheme: Scheme) where
+        Scheme.Node == Node,
+        Scheme.Weight == Weight
+    {
+        for edge in edges {
```

Comment:
Nit: I generally like the "cuddled" `else`:

```Swift
if let scalar = weightScheme.weight(from: edge.a, to: edge.b) {
    updateEdge(edge) { $0 * scalar }
else {
    remove(edge)
}
```

Reply:
(As it tightens and aligns the two conditional operations.)

Reply:
Presume you mean 
```swift
if let scalar = weightScheme.weight(from: edge.a, to: edge.b) {
    updateEdge(edge) { $0 * scalar }
} else {
    remove(edge)
}
```
which I am on board with too!

Reply:
“Do as I mean, not as I say” 🤦🏻‍♂️

Reply:
haha!